optimize/util/optimize-commons/src/main/resources/service-config.yaml

```diff
@@ -435,6 +435,8 @@ opensearch:
 # a connection to it.
 es:
   connection:
+    clusterTaskCheckingEnabled: ${CAMUNDA_OPTIMIZE_ELASTICSEARCH_CLUSTER_TASK_CHECKING_ENABLED:true}
```

Comment:
```suggestion
    #  Used to determine whether Optimize should attempt to check cluster tasks. This can be used in case Optimize does not have permission to query cluster tasks.
    clusterTaskCheckingEnabled: ${CAMUNDA_OPTIMIZE_ELASTICSEARCH_CLUSTER_TASK_CHECKING_ENABLED:true}
```

Reply:
fixed